content/en/blog/_posts/2022-03-31-Kubernetes-1-24-deprecations-and-removals

```diff
@@ -0,0 +1,90 @@
+---
+layout: blog
+title: "Kubernetes Removals and Deprecations In 1.24"
+date: 2022-03-31
```

Comment:
This date and the filename should be renamed to a future date
April 4 is just a suggestion if the changes are committed and reviewed today or on Monday, April 3

```suggestion
date: 2022-04-04
```

Reply:
Good suggestion @reylejano. I'll hold off on updating the date until we get closer to getting this approved.